src/AutoRest.CSharp/LowLevel/Output/LowLevelOutputLibraryFactory.cs

```diff
@@ -161,6 +161,55 @@ public static void SetRequestsToClients(IEnumerable<ClientInfo> clientInfos)
             {
                 foreach (var operation in clientInfo.Operations)
                 {
+                    if (operation.Requests.All(r => operation?.RequestMediaTypes?.Values.Contains(r) == true))
```

Comment:
We should not modify `CodeModel` tree. While this practice is popular in mgmt plane, I don't want to move it into the DPG. `CodeModel` tree must remain unchanged source of truth.